src/LocalAgent/Dockerfile_

```diff
@@ -32,13 +32,16 @@ ENV MINDARO_BUILD_NUMBER=${MindaroBuildNumber}
 RUN dotnet publish -c ${Configuration} -o /src/publish
 
 # Final container
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine3.12 as final
+FROM mcr.microsoft.com/dotnet/aspnet:6.0-alpine3.16 as final
 
 # Setup common tools
-COPY /build/setup-alpine-tools.sh .
-RUN chmod +x ./setup-alpine-tools.sh \
-    && ./setup-alpine-tools.sh \
-    && rm -f ./setup-alpine-tools.sh
+RUN apk update
+RUN apk upgrade
+RUN apk add -u bash
+RUN apk add -u procps
+RUN apk add -u curl
+RUN apk add -u bind-tools
+RUN apk add -u kubectl --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
```

Comment:
💡☕️ observation: we are adding kubectl, other tools for latter use? And previous setuptool mechanism was doing something different? Just wondering if this is improvement over dotnet6 refactor.

Reply:
this is reverted back.